packages/bridge-ui/src/App.svelte

```diff
@@ -54,66 +34,6 @@
   import type { RelayerAPI } from './domain/relayerApi';
   import { relayerApi, relayerBlockInfoMap } from './store/relayerApi';
 
-  const providerMap: Map<number, ethers.providers.JsonRpcProvider> = new Map<
-    number,
-    ethers.providers.JsonRpcProvider
-  >();
-
-  providerMap.set(
-    CHAIN_ID_MAINNET,
-    new ethers.providers.JsonRpcProvider(import.meta.env.VITE_L1_RPC_URL),
-  );
-  providerMap.set(
-    CHAIN_ID_TAIKO,
-    new ethers.providers.JsonRpcProvider(import.meta.env.VITE_L2_RPC_URL),
-  );
-  providers.set(providerMap);
-
-  const { chains: wagmiChains, provider } = configureChains(
-    [mainnet, taiko],
-    [
-      publicProvider(),
-      jsonRpcProvider({
-        rpc: (chain) => ({
-          http: providerMap.get(chain.id).connection.url,
-        }),
-      }),
-    ],
-  );
-
-  $wagmiClient = createClient({
-    autoConnect: true,
-    provider,
-    connectors: [
-      new MetaMaskConnector({
-        chains: wagmiChains,
-      }),
-      new CoinbaseWalletConnector({
-        chains: wagmiChains,
-        options: {
-          appName: 'Taiko Bridge',
-        },
-      }),
-      new WalletConnectConnector({
-        chains: wagmiChains,
-        options: {
-          qrcode: true,
-        },
-      }),
-    ],
-  });
-
-  const prover: Prover = new ProofService(providerMap);
-
-  const ethBridge = new ETHBridge(prover);
-  const erc20Bridge = new ERC20Bridge(prover);
-
-  bridges.update((store) => {
```

Comment:
We have same situation with this map. It's added into a store variable, when it could simply be a service. Again, I don't think this is part of the state of the application.

cc: @shadab-taiko @cyberhorsey